client/verta/verta/_internal_utils/_utils.py

```diff
@@ -361,7 +362,10 @@ def get_default_workspace(self):
         if workspace_id:
             return self.get_workspace_name_from_id(workspace_id)
         else:  # old backend
-            return self.get_personal_workspace()
+            if self._PERMISSION_V2:
```

Comment:
Honestly, I think we could just remove this entire `if-else`. And instead, we can simply `return self.get_workspace_name_from_id(workspace_id)`.

The `else:  # old backend` is for before January 2020, so I'd say it's reasonable to remove. And then we can also remove `def get_personal_workspace()`

Reply:
removed